### PR TITLE
Add python examples

### PR DESCRIPTION
**Pull request by daltzctr**
I did opt to not change the naming for variables, since it makes it hard to compare when swapping between the languages. I also only prefaced objects that should belong to the class with `self`

### CODE REVIEW

File: source/docs/api-reference/api-usage/actuator-limits.rst [hunk 1]
```diff
@@ -33,3 +33,15 @@ The state of the forward or reverse limit switch can be retrieved from the API v
          if (forwardLimit.GetValue() == signals::ForwardLimitValue::ClosedToGround) {
             // do action when forward limit is closed
          }
+
+   .. tab-item:: Python
+      :sync: Python
+
+      .. code-block:: python
+
+         import phoenix6
```
Comment (bhall-ctre) [inline source/docs/api-reference/api-usage/actuator-limits.rst:42]: FYI, there's going to be changes to this in beta-5. Python is going to follow the same submodule structure as C++. So in this case, we would need:
```py
from phoenix6 import signals.
# ...
if forward_limit.value is signals.ForwardLimitValue.CLOSED_TO_GROUND:
```
Also, all our Python examples use `snake_case`, we don't need `m_`, and if statements don't need parentheses. (see https://github.com/CrossTheRoadElec/Phoenix6-Examples/blob/python_examples/python/ArcadeDrive/robot.py)

**Review by bhall-ctre — CHANGES_REQUESTED** [review]

File: source/docs/api-reference/api-usage/actuator-limits.rst (hunk 1 above)
Comment (daltzctr) [reply source/docs/api-reference/api-usage/actuator-limits.rst:42]: In that case, does C++ also need to get refactored to follow C++ conventions?

File: source/docs/api-reference/api-usage/actuator-limits.rst [hunk 2]
```diff
@@ -40,8 +40,9 @@ The state of the forward or reverse limit switch can be retrieved from the API v
       .. code-block:: python
 
          import phoenix6
+         from phoenix6 import signals
 
-         forwardLimit = self.m_motor.get_forward_limit()
+         forward_limit = self.motor.get_forward_limit()
 
-         if (forwardLimit.value is phoenix6.CLOSED_TO_GROUND):
+         if (forward_limit.value is signals.CLOSED_TO_GROUND):
```
Comment (bhall-ctre) [inline source/docs/api-reference/api-usage/actuator-limits.rst:47]: Shouldn't you still need `signals.ForwardLimitValue.CLOSED_TO_GROUND`? I don't think enums are promoted to the global space, and I don't think we'd want to encourage using them like that anyways.

File: source/docs/api-reference/device-specific/talonfx/open-loop-requests.rst [hunk 3]
```diff
@@ -5,13 +5,19 @@ Open-Loop control typically refers to directly controlling device output.
 
 There are open-loop control requests for all TalonFX control output types. With the exception of FOC-only control requests, all open-loop control requests follow the naming pattern ``{ControlOutputType}Out``. For example, the open-loop Voltage control request is called ``VoltageOut``. FOC-only control requests follow the naming pattern ``{ControlOutputType}``.
 
+In the below example, note that devices are initialized with two arguments. These arguments correspond to :ref:`the device ID <docs/tuner/device-details-page:configuring name & ids>` and CAN bus name. For CANivore, this is the name of the CANivore as configured in :doc:`/docs/tuner/index`. For more information, see the :doc:`CANivore API </docs/canivore/canivore-api>` documentation.
```
Comment (bhall-ctre) [inline source/docs/api-reference/device-specific/talonfx/open-loop-requests.rst:8]: I'm not sure if this is the right place to put this. Maybe put this in API Overview? Or we actually create a page documenting creating devices. We need somewhere to document thread safety anyways.

**Review by bhall-ctre — APPROVED** [review]

File: source/docs/api-reference/device-specific/talonfx/open-loop-requests.rst (hunk 3 above)
Comment (bhall-ctre) [reply source/docs/api-reference/device-specific/talonfx/open-loop-requests.rst:8]: This can be changed in a separate PR.

**Pull request merged**